src/Util/Reflection.php

```diff
@@ -37,6 +37,8 @@ public function getProperty($methodName)
         if (preg_match('/^('.$pattern.')(.+)$/i', $methodName, $matches)) {
             return $matches[2];
         }
+        
+        return null;
```

Comment:
I agree with @teohhanhui to follow Symfony CS strictly. It also allows to run tools like PHP CS Fixer with the Symfony level.

Btw, I'm :+1: for symfony/symfony#17201. If the Symfony core team adopts it, we will follow. But while it's still a pending decision, keep it as is.